modules/backend/widgets/Form.php

```diff
@@ -1227,7 +1227,11 @@ public function getSaveData(): array
             // Exclude fields that didn't provide any value
             $fieldValue = $this->dataArrayGet($result, $parts, FormField::NO_SAVE_DATA);
             if ($fieldValue === FormField::NO_SAVE_DATA) {
-                continue;
+                if ($widget->getConfig('type') === 'taglist') {
+                    $fieldValue = in_array($widget->getConfig('mode'), ['array','relation']) ? [] : '';
+                } else {
+                    continue;
+                }
```

Comment:
The above code snippet should not be required if we can fix the TagList widget to include empty array or empty string in the POST data.

@bennothommo any help would be appreciated.

Reply:
fixed.